apps/client/src/pages/onboarding/components/button/button.css.ts

```diff
@@ -0,0 +1,60 @@
+import { themeVars } from '@bds/ui/styles';
+import { style } from '@vanilla-extract/css';
+import { recipe } from '@vanilla-extract/recipes';
+
+export const buttonVariants = recipe({
+  base: {
+    borderRadius: '12px',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    cursor: 'pointer',
+    transition: 'background 0.2s',
+    gap: '0.1rem',
+  },
+  variants: {
+    selected: {
+      true: {
+        backgroundColor: themeVars.color.primary100,
+        color: themeVars.color.primary600,
+        border: `1px solid ${themeVars.color.primary500}`,
+      },
+      false: {
+        backgroundColor: 'transparent',
+        color: themeVars.color.gray500,
```

Comment:
```suggestion
        color: themeVars.color.gray600,
```
정보입력 뷰 중, 건강사항 부분에서 gray600이라는 사실,,, 
근데, 남성/여성과 예/아니오 버튼의 색상은 gray400과 primary 500이라 피그마에 comment 달아뒀습니다 ! 
답변 확인하고 해당 컬러로 전체 변경해주시면 될 것 같아요 ㅎㅎ

Reply:
lg 버튼은 아예 없애고 sm 버튼은 gray600로 수정했습니다!